packages/reload-middleware/README.md

```diff
@@ -0,0 +1,68 @@
+#  `@sap-ux/reload-middleware`
+
+The `@sap-ux/reload-middleware` is a [Custom UI5 Server Middleware](https://sap.github.io/ui5-tooling/pages/extensibility/CustomServerMiddleware) which creates a websocket server upon the current running UI5 Server and also creates a filesystem watcher that listens for filesystem changes to the current project's flex changes files.
+
+This allows to create websocket connections to the current UI5 server and receive notifications through it about updated, created, or deleted flex change files.
+
+The middleware is intended to be used as an external file changes notification tool for Control Property Editor to notify the user about external manual changes to flex changes files and provide an ability to manually reload app preview editor after them.
+
+
+## Configuration Options
+| Option                 | Type      | Default Value    | Description                                                                                                                         |
+| ---------------------- | --------- | ---------------- | ----------------------------------------------------------------------------------------------------------------------------------- |
+| `debug`                | `boolean` | `false`          | Enables debug output                                                                                                                |
+
+## Usage
+The middleware can be used without configuration. However, since the middleware intercepts a few requests that might otherwise be handled by a different middleware, it is recommended to run it before other file serving middlewares e.g. `backend-proxy-middleware` and `ui5-proxy-middleware` (and the corresponding middlewares in the `@sap/ux-ui5-tooling`).
+
+
+### Minimal Configuration
+With no configuration provided, the local FLP will be available at `/test/flp.html` and the log level is `info`.
```

Comment:
That looks like copied and pasted from the `preview-middleware` readme. modify this text.